tests/env.rc.in

```diff
@@ -2,34 +2,34 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 
-PATH=@bindir@:@sbindir@:$PATH
+PATH=@bindir@:@sbindir@${PATH:+:${PATH}}
 export PATH
 
 GLUSTERD_PIDFILEDIR=@localstatedir@/run/gluster
 export GLUSTERD_PIDFILEDIR
 
-LD_LIBRARY_PATH=@libdir@:$LD_LIBRARY_PATH
+LD_LIBRARY_PATH=@libdir@${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 export LD_LIBRARY_PATH
 
-LIBRARY_PATH=@libdir@:$LIBRARY_PATH
+LIBRARY_PATH=@libdir@${LIBRARY_PATH:+:${LIBRARY_PATH}}
 export LIBRARY_PATH
 
-CPATH=@includedir@:$CPATH
+CPATH=@includedir@${CPATH:+:${CPATH}}
 export CPATH
 
 GLUSTERD_WORKDIR=@GLUSTERD_WORKDIR@
 export GLUSTERD_WORKDIR
 
-PKG_CONFIG_PATH=@pkgconfigdir@:$PKG_CONFIG_PATH
+PKG_CONFIG_PATH=@pkgconfigdir@${PKG_CONFIG_PATH:+:${PKG_CONFIG_PATH}}
 export PKG_CONFIG_PATH
 
-PYTHONPATH=@BUILD_PYTHON_SITE_PACKAGES@:$PYTHON_PATH
+PYTHONPATH=@BUILD_PYTHON_SITE_PACKAGES@${PYTHON_PATH:+:${PYTHON_PATH}}
 export PYTHONPATH
```

Comment:
Are these redundant because it seems to be defined again in line 32-33?

Reply:
You are right. This seems duplicate. I'll remove one of them.

It also seems that `PYTHON_PATH` is incorrect. I would say the good name is `PYTHONPATH` without `_`.